x-pack/test_serverless/functional/test_suites/security/platform_security/roles_management_card.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const testSubjects = getService('testSubjects');
+  const pageObjects = getPageObjects(['svlCommonPage', 'common']);
+  const browser = getService('browser');
+  const retry = getService('retry');
+
+  describe('Roles management card', function () {
+    this.tags('smoke');
+    before(async () => {
+      // Navigate to the index management page
+      await pageObjects.svlCommonPage.login();
+      await pageObjects.common.navigateToApp('management');
+    });
+
+    after(async () => {
+      await pageObjects.svlCommonPage.forceLogout();
```

Comment:
__question:__ instead of `login` and `forceLogout`, would it be possible to use `loginAsAdmin` (based on SAML) introduced in https://github.com/elastic/kibana/pull/176221/?

Reply:
Yes! That is a great idea.

Reply:
Fixed in latest commit